druid-shell/examples/hello.rs

```diff
@@ -109,10 +109,22 @@ fn main() {
     druid_shell::init();
 
     let mut file_menu = Menu::new();
-    file_menu.add_item(0x100, "E&xit", MenuKey::std_quit());
-    file_menu.add_item(0x101, "O&pen", MenuKey::command('o'));
+    file_menu.add_item(
+        0x100,
+        "E&xit",
+        Some(&HotKey::new(SysMods::Cmd, "q")),
+        true,
+        false,
+    );
+    file_menu.add_item(
+        0x101,
+        "O&pen",
+        Some(&HotKey::new(SysMods::Cmd, "o")),
+        true,
+        false,
+    );
     let mut menubar = Menu::new();
-    menubar.add_dropdown(file_menu, "&File");
+    menubar.add_dropdown(file_menu, "&File", true);
```

Comment:
For some reason, this menu doesn't seem to be showing up on mac (but is fine on Windows). I haven't investigated more deeply.

Reply:
It seems like the first menu item is always given the title of the running application; we would need to add an extra menu item if we wanted a file menu.